src/lib/Recoil_recoverableViolation.js

```diff
@@ -17,8 +17,10 @@ function recoverableViolation( // @oss-only
   message: string, // @oss-only
   projectName: 'recoil', // @oss-only
   {error}: {|error?: Error|}, // @oss-only
-): null { // @oss-only
-  if (__DEV__) { // @oss-only
+): null {
+  // @oss-only
```

Comment:
This is going to be a problem unfortunately, for the sync script to work (it's not right now), we need to have `// @oss-only` on every single line.

If you can add `// prettier-ignore` at the beginning of the function that would be great